packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6183,6 +6185,27 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.controlsOptionsManager.setDisallowUnmute(enabled);
   }
 
+  /**
+   * set the mute all flag for participants if you're the host
+   * @returns {Promise}
+   * @param {boolean} mutedEnabled
+   * @param {boolean} disallowUnmuteEnabled
+   * @param {boolean} muteOnEntryEnabled
+   * @public
+   * @memberof Meeting
+   */
+  public setMuteAll(
```

Comment:
what about setUnmuteAll?

Reply:
I followed the same pattern as disallowUnmute and muteOnEntry. mutedEnabled=true means mute all, mutedEnabled=false means unmute all

Reply:
ah got it